internal/config/settings.go

```diff
@@ -295,6 +310,7 @@ func InitGlobalSettings() error {
 // based on whether the path matches a glob
 // Must be called after ReadSettings
 func UpdatePathGlobLocals(settings map[string]interface{}, path string) {
+	checkDeprecatedSettings(settings)
```

Comment:
Do we really need to check this here once again?
We've validated and maybe extended the settings (even the maps) in `validateParsedSettings()` already.